pkg/security/security_profile/dump/activity_dump.go

```diff
@@ -8,7 +8,7 @@
 
 //go:generate go run github.com/mailru/easyjson/easyjson -gen_build_flags=-mod=mod -no_std_marshalers -build_tags linux $GOFILE
 
-package activitydump
+package dump
```

Comment:
All those file moves will create a lot of churn and conflict, can't we do that in a separate PR ?

Reply:
I _**think**_ we're good: the "only" changes we're expecting to make in the coming weeks are related to Security Profiles and will require at least this PR (which I hope we can merge soon). TL;DR I'll make another PR if we get just one conflict ! Deal ?